typescript-timers/timeout.ts

```diff
@@ -0,0 +1,5 @@
+const $h1 = document.querySelector('.message');
+if (!$h1) throw new Error('$h1 query has failed');
+setTimeout(() => {
+  $h1.innerHTML = 'Hello There';
```

Comment:
Should be using `textContent`. `innerHTML` should generally be avoided because it can lead to vulnerability attacks